### DISCUSSION TITLE
How do I deep link to a property of a c# class in docfx markdown?

### DISCUSSION THREAD

**Discussion by tig** [dotnet/docfx#10216] [Q&A]
e.g. 

```cs
/// <summary>A single tab in a <see cref="TabView"/>.</summary>
public class Tab : View
{
    private string _displayText;

    /// <summary>Creates a new unnamed tab with no controls inside.</summary>
    public Tab ()
    {
        BorderStyle = LineStyle.Rounded;
        CanFocus = true;
        TabStop = TabBehavior.NoStop;
    }

    /// <summary>The text to display in a <see cref="TabView"/>.</summary>
    /// <value></value>
    public string DisplayText
    {
        get => _displayText ?? "Unnamed";
        set
        {
            _displayText = value;
            SetNeedsDisplay ();
        }
    }

    /// <summary>The control to display when the tab is selected.</summary>
    /// <value></value>
    public View View { get; set; }
}
```

In `tab.md`

```md
## The @Terminal.Gui.Tab View

blah, blah, Use the `DisplayText` property (@Terminal.Gui.Tab.DisplayText) to...

```

I've tried:

* at Terminal.Gui.Tab.DisplayText: `@Terminal.Gui.Tab.DisplayText` - Just renders `@Terminal.Gui.Tab.DisplayText`

* xref: `[](xref:Terminal.Gui.Tab.DisplayText)` - nothing

* standard: [Terminal.Gui.Tab.DisplayText](~/api/Terminal.Gui.Tab.DisplayText.yml) - works but generates a warning:

```
 warning InvalidFileLink: Invalid file link:
```

This does, but is clunky and hard to maintain: `[Tab.DisplayText](~/api/Terminal.Gui.Tab.yml#Terminal_Gui_Tab_DisplayText)`

I really would l like to use either the `@` or `[](xref:)` syntax.

**Comment by KalleOlaviNiemitalo** [comment]
Does `<xref:Terminal.Gui.Tab.DisplayText>` work?

**Comment by tig** [comment]
I figured it out. The docs here are not great, FWIW: https://dotnet.github.io/docfx/docs/links-and-cross-references.html#more-options-for-cross-referencing-net-api-docs

* `@Terminal.Gui.View.Add(View)` works (method)
* `@Terminal.Gui.View.Subviews` works (property)
* `[](xref:Terminal.Gui.View.Add(View)` works

I found this doesn't:

```md
* **@Terminal.Gui.View.Add(View)** - test
```

Have to do this:

```md
* **@"Terminal.Gui.View.Add(View)"** - test
```

I can't get this to work:

```md
@"Terminal.Gui.View.Add(View)?text=View.Add()"
```
